web/crux-ui/e2e/with-login/nodes-deploy.spec.ts

```diff
@@ -12,7 +12,7 @@ import {
 
 const PREFIX = 'first'
 const PREFIX_TWO = 'second'
-const IMAGE = 'nginx'
+const IMAGE = NGINX_TEST_IMAGE_WITH_TAG
```

Comment:
In this case there's no need for `IMAGE`, it can be replaced with `NGINX_TEST_IMAGE_WITH_TAG`.